### PR TITLE
Console app and bff changes related to subscriber flows

### PR DESCRIPTION
**Pull request by salman-codewar**

### CODE REVIEW

File: interfaces/console/src/app/configure/node/[id]/site/[name]/install/page.tsx [hunk 1]
```diff
@@ -39,7 +39,8 @@ const SiteConfigure = ({ params }: ISiteConfigure) => {
   const searchParams = useSearchParams();
   const qpPower = searchParams.get('power') ?? '';
   const qpSwitch = searchParams.get('switch') ?? '';
-  const stepTracker = searchParams.get('step') ?? '1';
+  const flow = searchParams.get('flow') ?? 'onb';
+  const step = parseInt(searchParams.get('step') ?? '1');
   const qpbackhaul = searchParams.get('backhaul') ?? '';
 
   const formik = useFormik({
```
Comment (Brackleycassinga) [inline interfaces/console/src/app/configure/node/[id]/site/[name]/install/page.tsx:46]: Why is this formik empty ?

File: systems/console-bff/common/utils/index.ts [hunk 2]
```diff
@@ -242,34 +240,40 @@ const getBaseURL = async (
...
-  if (store) {
-    const baseURL = await getFromStore(store, `${orgName}-${sysName}`);
-    if (baseURL) {
-      logger.info(
-        `Base URL found in store for ${orgName}-${sysName}: ${baseURL}`
-      );
+  logger.info(`${store.get("org")}`);
+  // if (store) {
+  //   const baseURL = await getFromStore(store, `${orgName}-${sysName}`);
+  //   if (baseURL) {
```
Comment (Brackleycassinga) [inline systems/console-bff/common/utils/index.ts:246]: Comments should be removed.

**Review by Brackleycassinga — APPROVED** [review]

File: interfaces/console/src/app/configure/node/[id]/site/[name]/install/page.tsx (hunk 1 above)
Comment (salman-codewar) [reply interfaces/console/src/app/configure/node/[id]/site/[name]/install/page.tsx:46]: Its not empty, I think you didn't expanded the bottom code.

**Review by Brackleycassinga — APPROVED** [review]

**Review by delvatt — APPROVED** [review]
Good Job man.  Sounds very rock solid :smile:

**Pull request merged**